python/iwp/data_loader.py

```diff
@@ -513,3 +515,64 @@ def add_timestep_as_coord( ds ):
                             preprocess=add_timestep_as_coord )
 
     return ds
+
+
+    def get_data_loader( dataset_path_pattern,
+                         time_indices,
+                         variables=[],
+                         xy_slice_indices=[],
+                         batch_size=1000,
```

Comment:
We'll want a smaller default, say in the 32-128 range, until we have more experience with the model architecture.  The architecture itself will drive what the batch size is, particularly the largest batch size target GPUs can hold in memory while training.